MODULE.bazel

```diff
@@ -74,10 +74,10 @@ bazel_binaries = use_extension(
     "bazel_binaries",
     dev_dependency = True,
 )
-bazel_binaries.download(version = "7.2.1")
+bazel_binaries.download(version = "7.4.0")
```

Comment:
It would be useful to be able to export this value so that we can make sure it's consistent with our `.bazelversion` but I'm not sure how to do that effectively with `bzlmod`.

Reply:
`bzlmod` allows regular variables within a single `MODULE.bazel` file, but you can't `load` something from elsewhere, and because the file name doesn't end with `.bzl`, you can't `load` constants from the module file into other files.

Reply:
Good question for @sitaktif / @shs96c.

Reply:
> bzlmod allows regular variables within a single `MODULE.bazel` file

Is there an example of how to do this elsewhere? I tried to `load` another `.bzl` file and that failed in the way that you've stated.